app/views/events/_applicants_overview.html.erb

```diff
@@ -53,11 +57,10 @@
 
   <div class="container-fluid no-padding">
     <div class="btn-group" role="group">
-      <%= link_to t('.accept_all'),
-          events_path, :class => 'btn btn-default'%>
-
-      <%= link_to t('.print_all'),
-                  print_applications_event_path(@event.id), :target => "_blank", :class => 'btn btn-default'%>
+      <% if (can? :edit_applicants, Event) && (@event.application_letters.count <= @event.max_participants) %>
```

Comment:
If I got you right you would like to have an variable `can_accept_all` or something like that which is set by the controller? I am not sure if this is a good idea. As view partial it could be also used in other event pages?